packages/css/src/components/grid/grid.scss

```diff
@@ -3,11 +3,37 @@
  * Copyright (c) 2023 Gemeente Amsterdam
  */
 
+@use "sass:list";
+@use "sass:map";
 @import "../../common/breakpoint";
+@import "../../common/size";
+
+$amsterdam-grid-sizes: ();
+$amsterdam-grid-allowed-sizes: ("sm", "md", "lg");
+$amsterdam-grid-gap-classes: (
+  "gap-vertical": "row-gap",
+);
+$amsterdam-grid-padding-classes: (
+  "padding-bottom": "padding-block-end",
+  "padding-top": "padding-block-start",
+  "padding-vertical": "padding-block",
+);
+
+// Filter sizes to sm, md, lg
+@each $size, $label in $amsterdam-sizes {
```

Comment:
Not a big fan of sass loops. It is not very readable and since there are only a couple of options I would prefer to have them all here.